### ISSUE TITLE
`git lfs track` does not track existing files in sub directories

### ISSUE THREAD

**Issue by larsxschneider** [github/git-lfs#1546]
### Setup

```
git-lfs/1.4.1 (GitHub; darwin amd64; go 1.7)
git version 2.10.0
macOS 10.11.6
```

### Steps to reproduce

```bash
# Create the repo and add some files to plain Git
$ git init .
$ mkdir subdir
$ echo "content1" >root.dat
$ echo "content2" >subdir/sub.dat
$ git add .
$ sleep 1   # <-- sleep is only required if you run this test as script!
$ git commit -m "add content to Git"

# Move files to Git LFS
$ git lfs track "/root.dat"
$ git lfs track "/subdir/sub.dat"
$ git add .
$ git commit -m "move content to LFS"

# Check results
$ git cat-file -p :root.dat
version https://git-lfs.github.com/spec/v1
oid sha256:47d741b6059c6d7e99be25ce46fb9ba099cfd6515de1ef7681f93479d25996a4
size 9

$ git cat-file -p :subdir/sub.dat
content2
```

`git lfs track` did not track the existing file in the sub directory. This seems to happen to all files that are located in sub directories and tracked individually with a leading slash (e.g. `/subdir/sub.dat`).

**Comment by ttaylorr** [comment]
Thanks for opening this! I'll put this on my to-do list.

**Comment by ttaylorr** [comment]
Fixed in #1563.
